src/solver/toolkit.c

```diff
@@ -2076,6 +2089,15 @@ EXPORT_TOOLKIT int swmm_getLinkPollut(int index, SM_LinkPollut type, double **po
                 *pollutArray = result;
                 *length = Nobjects[POLLUT];
             } break;
+            case SM_LINKREACTORC:
+            {
+                for (p = 0; p < Nobjects[POLLUT]; p++)
+                {
+                    result[p] = Link[index].reactorQual[p];
+                }
+                *pollutArray = result;
+                *length = Nobjects[POLLUT];
```

Comment:
Same comment as with the node getter, moving the *length assignment out of the switch statement could make this more concise.